.github/workflows/publish-release-artifacts.yml

```diff
@@ -0,0 +1,180 @@
+name: Publish Release Artifacts
+
+on:
+  workflow_dispatch:
+    inputs:
+      ballerina_version:
+        description: 'Ballerina Version'
+        default: '2201.1.1'
+        required: true
+      short_version:
+        description: 'Short Version'
+        default: '2201.1.1'
+        required: true
+      display_version:
+        description: 'Display Version'
+        default: '2201.1.1 (Swan Lake Update 1)'
+        required: true
+      code_name:
```

Comment:
This is also should be able to removed if we short version thing

Reply:
@keizer619 do you mean to generate display version using version?